process/project_proposals.adoc

```diff
@@ -1,7 +1,92 @@
-== CNCF Project Proposal Process v1.2
+== CNCF Project Proposal Process v1.3
 
- . *Introduction*. This governance policy sets forth the proposal process for projects to be accepted into the Cloud Native Computing Foundation (“CNCF”). The process is the same for both existing projects which seek to move into the CNCF, and new projects to be formed within the CNCF.
- . *Project Proposal Requirements*. Projects must be proposed via https://github.com/cncf/toc/tree/master/proposals[GitHub]. Project proposals submitted to the CNCF (see https://github.com/cncf/toc/blob/master/proposals/kubernetes.adoc[example]) can be written in https://www.markdownguide.org[Markdown], http://asciidoc.org[AsciiDoc], or http://docutils.sourceforge.net/rst.html[reStructuredText] and must provide the following information to the best of your ability:
+=== Introduction 
+
+This governance policy sets forth the proposal process for projects to be accepted into the Cloud Native Computing Foundation (“CNCF”). The process is the same for both existing projects which seek to move into the CNCF, and new projects to be formed within the CNCF.
+
+=== Project stages
+
+image::project-stages.png[Project stages]
+
+=== Sandbox process
+
+All exceptions (and "no" outcomes) are handled by the TOC. Possible "no" outcomes include "not at this time", and the project may be encouraged to re-apply after addressing issues. Timeframes are approximate, to set expectations.
+
+image::sandbox-process.png[Sandbox process]
+
+. *Project Proposal*
+   * Project proposed through a https://github.com/cncf/toc/issues[GitHub issue]
+   * Template TBD
```

Comment:
here is the template: https://github.com/cncf/toc/blob/master/process/project_proposals.adoc

Reply:
oh... this is an editing of the old doc. i guess we should just move https://github.com/cncf/toc/pull/355/files#diff-92faef9f43706befa4fdea2c30f0685bR91-R109 to a new file, and use it as a proposal template.

Reply:
There is also https://github.com/cncf/toc/issues/344 in flight. I'd suggest we get this process merged and then we can consolidate on a proposal template as a next step

Reply:
ok. makes sense. 

#344 mainly wants to solve

> Primary aim: This template helps a project to prepare for a presentation to the SIG, and then helps the SIG share the collated information with the TOC.

But I agree with you on consolidation. We should just merge that proposed template with the original project proposal templated since there are a lot of overlappings. Maintainers do not need to create two docs: one for github issue record, one for the sig information collecting.